/**:
  ros__parameters:
    type: 'mcbl2805'
    device: '/dev/motor_z'
    baud: 19200
    timeout: 0.2
    update_period_ms: 100
    # 1000 increments per rev (see datasheet)
    increments_per_rev: 1000
    increments_per_length_unit: 2043204
    nominal_rpm: 7700
